File: HW1_C++/Ex10.cpp
#include <cstdlib>
#include <ctime>
#include <iostream>
using namespace std;
int main()
{
    int secret_number = rand() % 100;
    int input = 0, cnt = 0;
    do
    {
        cin >> input;
        if (input > secret_number)
            cout << "It is too large" << endl;
        else if (input < secret_number)
            cout << "It is too small" << endl;
        cnt++;
    } while (secret_number != input);
    cout << "You won this game and you have " << cnt << " tries!" << endl;
}

File: HW2_Python/Ex10.py
import random
secret_number = random.randint(1,1000)
counter = 0
while True:
    entery_number = int(input())
    if entery_number == secret_number:
        print("You won this game!")
        break
    elif entery_number < secret_number:
        print("It is too small")
    else:
        print("It is too big")
    counter += 1

print(f"{counter} tries")